bolt/include/bolt/Core/ParallelUtilities.h

```diff
@@ -50,7 +50,7 @@ enum SchedulingPolicy {
 };
 
 /// Return the managed thread pool and initialize it if not initialized.
-ThreadPoolInterface &getThreadPool();
+ThreadPoolInterface &getThreadPool(const int ThreadsCount = opts::ThreadCount);
```

Comment:
@ayermolo We talked about setting the default value to min(ops::ThreadCount, DebugThreadCount) but that wouldn't work since DebugThreadCount is initialized in DWARFRewriter.cpp. Also wouldn't there be scenarios where we don't want the default to be the minimum of the two?

Reply:
I meant not for default, but at call site in DWARFRewriter.cpp

Reply:
> I meant not for default, but at call site in DWARFRewriter.cpp

So for the default opt value, correct? Instead of cl::init(1).

Reply:
I mean here:
ThreadPoolInterface &ThreadPool =
        ParallelUtilities::getThreadPool(opts::DebugThreadCount);